src/ucs/arch/x86_64/cpu.h

```diff
@@ -85,6 +88,27 @@ static inline void ucs_arch_clear_cache(void *start, void *end)
 }
 #endif
 
+static inline void *ucs_memcpy(void *dst, const void *src, size_t len) {
+#if defined(ENABLE_BUILTIN_MEMCPY)
+    if (ucs_unlikely((len > ucs_global_opts.memcpy_min_thresh) &&
+                     (len < ucs_global_opts.memcpy_max_thresh))) {
+        asm volatile ("rep movsb"
+                      : "=D" (dst),
+                      "=S" (src),
+                      "=c" (len)
+                      : "0" (dst),
+                      "1" (src),
+                      "2" (len)
+                      : "memory");
+        return dst;
+    } else {
```

Comment:
i think we can just remove this else and "fall through" to `#else`

Reply:
yep, simplified